docs/framework.md

```diff
@@ -0,0 +1,49 @@
+# ASP.NET Framework Configuration
+
+In order to setup the adapters for usage within an ASP.NET framework app, a module is inserted when the package is installed:
+
+```xml
+<configuration>
+  ...
+  <system.webServer>
+    <modules>
+      <remove name="SystemWebAdapterModule" />
+      <add name="SystemWebAdapterModule" type="Microsoft.AspNetCore.SystemWebAdapters.SystemWebAdapterModule, Microsoft.AspNetCore.SystemWebAdapters" preCondition="managedHandler" />
+    </modules>
+  </system.webServer>
+  ...
+</configuration>
+```
+
+This module will only run when in the context of a managed application and only if configured to do so. This configuration should take place in `Global.asax.cs` or `Global.asax.vb` and would look like the following:
+
+```cs
+protected void Application_Start()
+{
+    AreaRegistration.RegisterAllAreas();
+    GlobalConfiguration.Configure(WebApiConfig.Register);
+    FilterConfig.RegisterGlobalFilters(GlobalFilters.Filters);
+    RouteConfig.RegisterRoutes(RouteTable.Routes);
+    BundleConfig.RegisterBundles(BundleTable.Bundles);
+
+    Application.AddSystemWebAdapters()
+        .AddProxySupport(options => options.UseForwardedHeaders = true)
+        .AddRemoteAppSession(
+            options => options.ApiKey = ClassLibrary.SessionUtils.ApiKey,
+            options => ClassLibrary.SessionUtils.RegisterSessionKeys(options));
+}
+```
+
+Customizing the adapters is done by adding modules to via `Application.AddSystemWebAdapters()` which will be loaded within the `SystemWebAdaptersModule` registered in `web.config`. The available configuration right now are described below.
+
+## Additional Modules
+
+### Proxy Support
+
+During migration, the framework app will be moved to be downstream from a reverse proxy. In order for things like port, url, and scheme to be constructed correctly, we must update the values `HttpContext` use. This is similar to the [process used on ASp.NET Core](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer).
```

Comment:
`During migration, the framework app will be moved to be downstream from a reverse proxy.`
could become
`During migration, the framework app will be moved and will be downstream from a reverse proxy.`